package.json

```diff
@@ -1,6 +1,6 @@
 {
   "name": "gaiden",
-  "version": "21.6.1",
+  "version": "22.0.0",
```

Comment:
The conflicts was caused because this change, but I consider it a breaking change, so this version makes sense, i guess.